table/server/common/src/main/java/alluxio/table/common/udb/PathTranslator.java

```diff
@@ -45,16 +48,7 @@ public PathTranslator() {
    * @return PathTranslator object
    */
   public PathTranslator addMapping(String alluxioPath, String ufsPath) {
-    while (alluxioPath.endsWith("/")) {
-      // strip trailing slashes
-      alluxioPath = alluxioPath.substring(0, alluxioPath.length() - 1);
-    }
-
-    while (ufsPath.endsWith("/")) {
-      // strip trailing slashes
-      ufsPath = ufsPath.substring(0, ufsPath.length() - 1);
-    }
-    mPathMap.put(alluxioPath, ufsPath);
+    mPathMap.put(new AlluxioURI(alluxioPath), new AlluxioURI(ufsPath));
```

Comment:
what is the reason to move this to the translation time? would this add potentially duplicate keys like `/path1/path2' and `/path1/path2/` ?

Reply:
> move this to the translation time

Do you mean the creation of `AlluxioURI`s? I simply use them for the convenient methods like `isAncestorOf` and that it trims trailing slashes already.

Reply:
oh somehow i missed that you create AlluxioURIs out of them, yeah this works